src/components/SellerPage/SellerPage.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import HeroBanner from '../HeroBanner';
+import Footer from '../Footer';
+import { sampleMerchant } from './sample-merchant';
+import { StoreInfo } from '../StoreInfo';
+import OwnerPanel from '../OwnerPanel';
+import styles from './styles.module.scss';
+import { getSeller } from '../../utilities';
+
+const SellerPage: React.FC<{}> = () => {
+  // fix typing
+  const [seller, setSeller] = useState<any | null>();
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    const result = await getSeller();
+    setSeller(result.data);
+  };
+
+  // TO DO: handle actual null states and loading
```

Comment:
@kevinjtsai FYI we'd need a null state page. Maybe we can ask Jess or Holly to render something for us?